resources/asciidoctor/lib/docbook_compat/convert_document.rb

```diff
@@ -58,13 +63,26 @@ def munge_body(doc, html)
 
     def munge_title(doc, html)
       id = doc.id || 'id-1'
-      header_start = <<~HTML.strip
-        <div class="titlepage"><div><div>
+      # Important: we're not replacing he whole header - it still will have a
```

Comment:
I started by trying to munge the toc out of where it was in the header and realized that a lot more thing can end up in the header. I ended up not trying to get the toc out with string manipulation but I kept this part of the change so we could handle other things in the header without errors.